test/AffInterface.purs

```diff
@@ -16,7 +16,6 @@ import Ctl.Internal.QueryM
 import Ctl.Internal.QueryM.CurrentEpoch (getCurrentEpoch)
 import Ctl.Internal.QueryM.EraSummaries (getEraSummaries)
 import Ctl.Internal.QueryM.Ogmios (OgmiosAddress)
-import Ctl.Internal.QueryM.ProtocolParameters (getProtocolParameters)
```

Comment:
No point in testing `getProtocolParameterAff, it's necessarily called to setup a contract, and getProtocolParameter is just asks, which doesn't need testing in the way we do here